cmd/backup-manager/app/util/remote.go

```diff
@@ -159,3 +229,23 @@ func checkS3Config(s3 *v1alpha1.S3StorageProvider, fakeRegion bool) *s3Query {
 
 	return &sqs
 }
+
+// checkS3Config constructs s3Query parameters
+func checkGcsConfig(gcs *v1alpha1.GcsStorageProvider, fakeRegion bool) *gcsQuery {
+	gqs := gcsQuery{}
+
+	gqs.bucket = gcs.Bucket
+	gqs.location = gcs.Location
+	gqs.path = gcs.Path
+	gqs.projectId = gcs.ProjectId
+	gqs.storageClass = gcs.StorageClass
+	gqs.objectAcl = gcs.ObjectAcl
+	gqs.bucketAcl = gcs.BucketAcl
+	gqs.secretName = gcs.SecretName
+	gqs.prefix = gcs.Prefix
```

Comment:
The `Location`, `Path`, `ProjectID`, `BucketAcl` and `SecretName` are not used?

Reply:
temporary not used